### DISCUSSION TITLE
GCam 8.2.4 crashing on Pixel 5 with GCam Services Provider 1.2 on GrapheneOS

### DISCUSSION THREAD

**Discussion by gryban1** [lukaspieper/Gcam-Services-Provider#52] [General]
Hi,

I have a new Pixel 5 with the latest GrapheneOS and the precompiled GCam Services Provider 1.2 installed via:
`adb push app-debug.apk /data/local/tmp`
`adb shell pm install --force-queryable /data/local/tmp/app-debug.apk`

When I now install the latest GCam for Pixel 5 (8.2.4) via Aurora Store and launch it, it normally starts and I can use it. But only this first time. When I stop the app and launch it again it instantly crashes.
When I clear the whole app data and restart the phone I can start the app again, but here again only one time.

Anyone having the same issue?
If you need any logs, can you please tell me where I can find and how I can provide it?

Best regards,
Roland

**Comment by gryban1** [comment]
The following behaviour is a stable workaround for the gcam 8.2.400:

Install googles app "photos"
Launch "photos"
now launch gcam
-> no crash and works

as long as you have photos launched in the background gcam works smoothly.
when you close photos in most cases you have to launch photos again before using gcam.

**Comment by lukaspieper** [comment]
Hi,

thanks for reaching out. Looks like this is the same issue mentioned [here](https://github.com/lukaspieper/Gcam-Services-Provider/discussions/17#discussioncomment-1038039).

While it is known that Gcam is hard wired to Google Photos and it's great that installing/opening worked for you, I cannot give any insights into this at the moment. I likely look into this once I got a new Pixel but currently it's difficult to get one and they are very expensive due to shortage.

**Comment by gryban1** [comment]
New gcam 8.4.300 is released for Pixel 5 with which the problem seems no more present.
I still have google photos installed. I don't know how the camera behaves without it.

**Reply by gryban1** [reply]
After some days of testing the google camera is now stable with this new version 8.4.300 on Pixel 5.
